README.md

```diff
@@ -49,7 +50,7 @@ LABEL discovery
 MENU LABEL Foreman Discovery Image
 MENU DEFAULT
 KERNEL boot/fdi-image/vmlinuz0
-APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=http://YOURPROXY proxy.type=proxy
+APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image rd.debug=1 acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=http://YOURPROXY proxy.type=proxy
```

Comment:
Yes, it will basically give you emergency shell instead of halting system when something bad happens. Better than halt... You can still investigate logs.